src/sentry/static/sentry/app/routes.jsx

```diff
@@ -996,6 +996,16 @@ function routes() {
         component={errorHandler(LazyLoad)}
       />
 
+      <Route
+        path="/organizations/:orgId/data-export/:dataExportId"
+        componentPromise={() =>
+          import(
+            /* webpackChunkName: "DataDownloadView" */ 'app/views/dataExport/dataDownload'
+          )
+        }
+        component={errorHandler(LazyLoad)}
+      />
+
```

Comment:
The route for the download page was moved outside of the LightWeightOrganizationDetails wrapper route, since it no longer has the sidebar, or footer. It's just a standalone page. If this isn't the right way to accomplish this please LMK!

Reply:
@leeandher how does a user go back to sentry?

Reply:
You can click the sentry-logo in the small sidebar to go to the `/` route, but besides that the Valid/Early pages don't really direct back to the proper site.

The Expired page links back to wherever the dataset came from (i.e. Issues or Discover)